docs/source/ipex/inference.mdx

```diff
@@ -0,0 +1,47 @@
+<!--Copyright 2024 The HuggingFace Team. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Inference
+
+Optimum Intel can be used to load models from the [Hub](https://huggingface.co/models) and create pipelines to run inference with IPEX optimization (including patching, weight prepack and graph mode) on a variety of Intel processors (currently only support for CPU)
+
+
+## Loading
+
+### Transformers models
+
+You can load models from HuggingFace Model Hub, it will be optimized by ipex (including patching, weight prepack and graph mode) during loading.
```

Comment:
I think it could make sense to discuss about what is patched instead of "including patching"

Reply:
Done, please take a review.